pkg/packagekit/updates.jsx

```diff
@@ -176,12 +176,16 @@ function HeaderBar(props) {
         return null;
 
     if (props.state == "available") {
-        state = cockpit.ngettext("$0 update", "$0 updates", num_updates);
         for (let u in props.updates)
             if (props.updates[u].security)
                 ++num_security;
-        if (num_security > 0)
-            state += cockpit.ngettext(", including $1 security fix", ", including $1 security fixes",  num_security);
+        if (num_updates == num_security)
+            state = cockpit.ngettext("$1 security fix", "$1 security fixes", num_security);
```

Comment:
That branch would also hit for 0 updates, wouldn't it? That would look a bit odd. I think an empty string would be fine in this case, the big "your system is up to date" empty state pattern is enough information.

Reply:
With 0 updates, the state isn't "available", so we don't end up here at all.

Reply:
Right, sorry about that. Seems I already dumped all my working knowledge about this from my brain :-)